DeDRM_plugin/config.py

```diff
@@ -431,16 +431,19 @@ def export_key(self):
         defaultname = "{0}.{1}".format(keyname, self.keyfile_ext)
         filename = choose_save_file(self, unique_dlg_name,  caption, filters, all_files=False, initial_filename=defaultname)
         if filename:
-            with file(filename, 'wb') as fname:
```

Comment:
If you want to keep the new code structure close to the old one, you could open the file as binary and inside that manually encode the text data.
```python
with open(fname, 'wb'):
     if self.binary_file:
       # should use codecs.decode for decoding with non-text encodings
        fname.write(codecs.decode(self.plugin_keys[keyname]), 'hex')
     elif self.json_file:
        ...
     elif self.android_file:
        for ... :
           fname.write(key.encode())
           fname.write(b"\n")
     else:
        fname.write(...)
```